reg_tests/executeOpenfastRegressionCase.py

```diff
@@ -121,8 +123,12 @@
 
 ### Run openfast on the test case
 if not noExec:
-    caseInputFile = os.path.join(testBuildDirectory, caseName + ".fst")
-    returnCode = openfastDrivers.runOpenfastCase(caseInputFile, executable)
+    if steadyState:
+        caseInputFile = os.path.join(testBuildDirectory, caseName + ".inp")
```

Comment:
I'd like to move away from the `.inp` extension, and focus on using `.dat`, `.dvr`, `.fst` and .`fstf` (which is already a lot). Could we use `.dvr` here?

Reply:
Done.